app/src/main/java/org/simple/clinic/editpatient/EditPatientViewRenderer.kt

```diff
@@ -21,6 +21,8 @@ class EditPatientViewRenderer(private val ui: EditPatientUi) : ViewRenderer<Edit
     ui.setDateOfBirthAndAgeVisibility(fieldToShow)
 
     manageButtonState(model)
+
+    model.colonyOrVillagesList?.let { ui.showColonyOrVillagesList(it) }
```

Comment:
Can we instead use an if condition along with computed properties in the model. Please refer to other ViewRenderer files

```kt
if (model.hasColonyOrVillagesList) {
  ui.showColonyOrVillagesList(model.colonyOrVillagesList!!)
}
```